report_template.ipynb

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
Do we want to create a `notebooks` folder to store `example.ipynb` and `report_template.ipynb` together?

Reply:
Let's do that on the next PR.